### PR TITLE
fix issues and add a feature

### PR DESCRIPTION
**Pull request by cameronmartino**
Version bumped from v0.0.4 -> v0.0.5

### Features
* The output of the standalone and QIIME2 -CLI now return a state ordination
    * Note that only the standalone can handle more than one state due to the lack of lists in QIIME2

### Bug fixes

* Fixes in `_transformer.py` and `ctf.py` to utalize QIIME2 metadata passing, this fixes the dtype bool dropping issues in metadata
* Removed imports of QIIME2 metadata in `ctf.py`, this will allow the standalone commandline to run without having QIIME2 installed.
* Update pandas future error of .loc[new_index, :] to .reindex(new_index)
* Centered the subject biplot
* Added flake8 testing to travis

### CODE REVIEW

**Comment by cameronmartino** [comment]
note this also closes #13, closes #6, closes #7, closes #8

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
-(2019-06-12)
+v0.0.5 (2019-06-12)
+
+### Features
+* The output of the standalone and QIIME2 -CLI now return a state ordination
+    * Note that only the standalone can handle more than one state due to the lack of lists in QIIME2
+
+### Bug fixes
+
+* Fixes in `_transformer.py` and `ctf.py` to utalize QIIME2 metadata passing, this fixes the dtype bool dropping issues in metadata
```
Comment (gwarmstrong) [inline CHANGELOG.md]: `utalize` -> `utilize`

File: gemelli/_ctf_defaults.py [hunk 2]
```diff
@@ -35,9 +35,17 @@
...
+QDIST = ("A sample-sample distance matrix generated from "
+         " the euclidean distance of the subject-state "
+         "ordinations and itself.")
 QLOAD = ("Compositional biplot of subjects as points and features as arrows."
-        " Where the variation between subject groupings is explained by the"
-        " log-ratio between opposing arrows. WARNING: The % variance explained"
-        " is spread over n_components and can be inflated.")
+         " Where the variation between subject groupings is explained by the"
+         " log-ratio between opposing arrows. "
+         "WARNING: The % variance explained is spread over n_components "
```
Comment (gwarmstrong) [inline gemelli/_ctf_defaults.py]: maybe you could put something like "only spread over n_components" to indicate that this is lower than nomral

File: gemelli/_ctf_defaults.py [hunk 3]
```diff
@@ -35,9 +35,17 @@
...
+         "and can be inflated.")
+QSOAD = ("Compositional biplot of states as points and features as arrows."
+         " Where the variation between subject groupings is explained by the"
+         " log-ratio between opposing arrows. "
+         "WARNING: The % variance explained is spread over n_components "
+         "and can be inflated.")
+OUT_PFAM = ()
```
Comment (gwarmstrong) [inline gemelli/_ctf_defaults.py]: I don't see this being used anywhere

File: gemelli/_ctf_defaults.py (hunk 2 above)
Comment (cameronmartino) [reply gemelli/_ctf_defaults.py]: done, thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (cameronmartino) [reply CHANGELOG.md]: done, thanks!

File: gemelli/_ctf_defaults.py (hunk 3 above)
Comment (cameronmartino) [reply gemelli/_ctf_defaults.py]: removed, thanks!

**Comment by cameronmartino** [comment]
@gwarmstrong this is a good point, I have added a test for center=True in `test_factorization.py`. Thanks!

**Pull request merged**